feature-detects/flash.js

```diff
@@ -77,11 +85,18 @@ define(['Modernizr', 'createElement', 'docElement', 'addTest', 'getBody'], funct
           }
         }
 
-        // If we’re rockin’ a fake body, clean it up
-        if (body.fake) {
+        // If we’re rockin’ an attached fake body, clean it up
+        if (body.fake && body.parentNode) {
```

Comment:
Confirmed—installing Modernizr at commit [https://github.com/Modernizr/Modernizr/commit/a6ca9d9f240071107249241f83223c4185151875](a6ca9d) works as normal, but Modernizr at commit [https://github.com/Modernizr/Modernizr/commit/1db6eee599e40848b5f571f3c09f32b64b926d82](1db6ee) leaves the extra body element in.